packages/react-examples/src/react-accordion/AccordionItem/AccordionItem.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import {
+  AccordionItem,
+  AccordionHeader,
+  AccordionPanel,
+  Accordion,
+  AccordionHeaderSize,
+  AccordionHeaderExpandIconPosition,
+} from '@fluentui/react-accordion';
+import { ChevronRightIcon, ChevronDownIcon } from '@fluentui/react-icons-mdl2';
```

Comment:
As we discussed in offline, let's have some default icons for slots in a package itself so it will not be required for consumers always specify icons.